packages/devtools_app/lib/src/shared/primitives/utils.dart

```diff
@@ -1167,7 +1167,7 @@ String simplifyDevToolsUrl(String url) {
   // http://localhost:123/#/page-id?key=value
   // Since we just want the query params, we will modify the url to have an
   // easy-to-parse form.
-  return url.replaceFirst(RegExp(r'#\/(\w*)[?]'), '?');
+  return url.replaceFirst(RegExp(r'#\/([\w\-]*)[?]'), '?');
```

Comment:
if we are changing this method, we should have a test that verifies this still gives us the expected behavior. The `app-size` screen is not the only page with a hyphen in the name, so I'm surprised that we haven't seen broken behavior before. So would be good to test this thoroughly.

Reply:
I just updated the test to iterate over all existing tabs